src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
```

Comment:
The constants 'cost', 'smallDiscount', and 'bigDiscount' should have more descriptive names. For example, 'DAILY_RENTAL_COST', 'DISCOUNT_FOR_THREE_DAYS', and 'DISCOUNT_FOR_SEVEN_DAYS'.